### PR TITLE
Smoke: Separated scripts for ShowConstantTBT and Slider from smoke API script

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Separated scripts for ShowConstantTBT and Slider from smoke API script

### CODE REVIEW

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  softButtons = requestParams.softButtons
+}
+
+local allParams = {
+  requestParams = requestParams,
+  responseUiParams = responseUiParams
+}
+
+--[[ Local Functions ]]
+local function ptuUpdateFuncRPC(tbl)
```
Comment (AByzhynar) [inline test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @GetmanetsIrina Why ptu update func - PTU already mean Policy Table Update - therefore looks that PTUpdateFuncRPC will be more correct

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    syncFileName = 'icon.png',
+    fileType = "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  navigationText1 ="navigationText1",
```
Comment (aderiabin) [inline test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @GetmanetsIrina Please add spaces after "=" in next few lines.

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+--[[ Local Functions ]]
+local function ptuUpdateFuncRPC(tbl)
+  local ShowCTBT = {
+    rpcs = {
+      ShowConstantTBT = {
+        hmi_levels = { "NONE", "BACKGROUND", "FULL", "LIMITED" }
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = ShowCTBT
```
Comment (aderiabin) [inline test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @GetmanetsIrina I would propose to use "point" notation for constant string keys in tables and "braces" notation for variable string keys and other types keys.
Examples:
```
tbl.policy_table.functional_groupings.NewTestCaseGroup = SLgroup
```
but
```
local groupName = "NewTestCaseGroup"
tbl.policy_table.functional_groupings[groupName ] = SLgroup
```

File: test_scripts/Smoke/API/017_Slider_PositiveCase.lua [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS", sliderPosition = 1 })
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonSmoke.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonSmoke.start)
+runner.Step("RAI, PTU", commonSmoke.registerApplicationWithPTU)
+runner.Step("Activate App", commonSmoke.activateApp)
+runner.Step("Upload icon file", commonSmoke.putFile, {putFileParams})
```
Comment (dboltovskyi) [inline test_scripts/Smoke/API/017_Slider_PositiveCase.lua]: @GetmanetsIrina Is it required to have icon file in order to test this RPC?

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua (hunk 1 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @AByzhynar , Done in d21fba6bc97103738bb5d7fa5a8905e3ade2cf39

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua (hunk 2 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @aderiabin , Done in d21fba6bc97103738bb5d7fa5a8905e3ade2cf39

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua (hunk 3 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @aderiabin , Done in d21fba6bc97103738bb5d7fa5a8905e3ade2cf39

File: test_scripts/Smoke/API/017_Slider_PositiveCase.lua (hunk 4 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/017_Slider_PositiveCase.lua]: @dboltovskyi , Step is redundant, removed in d21fba6bc97103738bb5d7fa5a8905e3ade2cf39

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  navigationText1 = "navigationText1",
+  navigationText2 = "navigationText2",
+  eta = "12:34",
+  totalDistance = "100miles",
```
Comment (AKalinich-Luxoft) [inline test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @GetmanetsIrina please add timeToDestination param value

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+      },
+      isHighlighted = true,
+      softButtonID = 44,
+      systemAction ="DEFAULT_ACTION",
+    },
+  },
+}
+
+local responseUiParams = {
+  navigationTexts = {
```
Comment (AKalinich-Luxoft) [inline test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @GetmanetsIrina please add timeToDestination param value

File: test_scripts/Smoke/API/017_Slider_PositiveCase.lua [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Variables ]]
+local requestParams = {
+  numTicks = 7,
+  position = 1,
+  sliderHeader ="sliderHeader",
```
Comment (AKalinich-Luxoft) [inline test_scripts/Smoke/API/017_Slider_PositiveCase.lua]: @GetmanetsIrina please add sliderFooter param value

File: test_scripts/Smoke/API/017_Slider_PositiveCase.lua [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+  local cid = self.mobileSession1:SendRPC("Slider", params)
+  params.appID = commonSmoke.getHMIAppId()
+  EXPECT_HMICALL("UI.Slider", params)
+  :Do(function(_,data)
+      self.hmiConnection:SendNotification("UI.OnSystemContext",{ appID = params.appID, systemContext = "HMI_OBSCURED" })
+      local function sendReponse()
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {sliderPosition = 1})
+        self.hmiConnection:SendNotification("UI.OnSystemContext",{ appID = params.appID, systemContext = "MAIN" })
+      end
+      RUN_AFTER(sendReponse, 1000)
```
Comment (AKalinich-Luxoft) [inline test_scripts/Smoke/API/017_Slider_PositiveCase.lua]: @GetmanetsIrina this hardcoded delay could be replaced with `params.timeout`

Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/017_Slider_PositiveCase.lua]: @AKalinich-Luxoft , time in RUN_AFTER is not related to timeout at all, this time is to perform some delay before sending response for emulating user action.  So rejected.

File: test_scripts/Smoke/API/017_Slider_PositiveCase.lua (hunk 7 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/017_Slider_PositiveCase.lua]: @AKalinich-Luxoft , Done in e5c425f45f8c09e764332a840527f30c5ab2c793.

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua (hunk 6 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @AKalinich-Luxoft , Done in e5c425f45f8c09e764332a840527f30c5ab2c793.

File: test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua (hunk 5 above)
Comment (GetmanetsIrina) [reply test_scripts/Smoke/API/016_ShowConstantTBT_PositiveCase.lua]: @AKalinich-Luxoft , Done in e5c425f45f8c09e764332a840527f30c5ab2c793.

**Pull request merged**